README.md

```diff
@@ -55,13 +55,13 @@ where argument, `dist`  is the directory to zip and `next-$(git rev-parse --shor
 The `md-to-html` script converts a given collection of markdown files to html. The script accepts 2 arguments in the below order respectively:
 | Argument | Description |
```

Comment:
It looks like there needs to be a line break before the table, it's not displayed correctly. (same for the table on line 38)